Gems/Atom/RHI/Code/Source/RHI/Device.cpp

```diff
@@ -75,6 +75,7 @@ namespace AZ
             m_nearestSupportedFormats.fill(Format::Unknown);
             
             m_physicalDevice = &physicalDevice;
+            m_index = index;
```

Comment:
Lets add an assert here.
AZ_Assert(index > 0, "Multi gpu support is not implemented yet.")